vignettes/c-advanced-base.Rmd

```diff
@@ -81,7 +81,25 @@ for (i in seq_along(species_split)) {
 }
 ```
 
-That's a nice basic plot! But you know what would make it nicer? If we used little penguins instead of points! To do that, we can use the `add_phylopic_base()` function. In this case, we'll specify `img = penguin_rot` since we want to use the same image for each x-y pair:
+That's a nice basic plot! But you know what would make it nicer? If we added a penguin silhouette to the plot. Sadly, we don't have a different silhouette for each species (although we could make one...), so let's just go with putting a single silhouette in the top panel. To do that, we can use the `add_phylopic_base()` function. Note that the `x` and `y` arguments specify the center of the silhouette, and the `ysize` argument specifies how tall the silhouette is in the units of the y-axis.
+
+```{r}
```

Comment:
Nice! It's a bit of a complicated example to show this off, but it is good that you've mimicked the ggplot tutorial, so I think it is fine.